build/config.props

```diff
@@ -31,9 +31,16 @@
     <TestSDKVersion>15.5.0</TestSDKVersion>
     <MoqVersion>4.10.1</MoqVersion>
     <FluentAssertionsVersion>4.19.4</FluentAssertionsVersion>
-    <MicrosoftBuildPackageVersion>15.1.1012</MicrosoftBuildPackageVersion>
+
+    <MicrosoftBuildFrameworkPackageVersion>16.0.461</MicrosoftBuildFrameworkPackageVersion>
```

Comment:
Why split up the variables if they are going to be the same? 
In fact we should strive to keep them the same, so I think we should prefer having a single variable. 

Any mixing and matching from packages build from the same repo is a likely untested matrix. While semver's guidance would say that's perfectly fine (and in most cases it is), we should still not complicate our lives with potentially having mismatching versions :)

Reply:
MSBuild's version aren't semver. Please don't mix them :)

Reply:
Ok, I will leave the MSBuild package versions in separated properties.

Reply:
I think our feedback was the exact opposite. 
Change it back to only 1 property. 
Rainer is suggesting to never have different versions of the MSBuild packages throughout our repo.

Reply:
@rainersigwald 

If/when you ship patches for MSBUild, do you build and release all the packages in repo? 

For example;
NuGet's patching model is release a new version of everything. 
ASP.NET's patching model is do the best to release a new version of only the packages changing and everything above them.

Reply:
We release a new version of everything.

Reply:
Discussed on chat/mail. We can have the same MicrosoftBuildPackageVersion property and dotnet source-build should be able to override it. We don't need a property for each MSBuild package version.